### PR TITLE
Rename `async` function and variable names (add support for python3.7)

### LINKED ISSUE #99: Build fails on python3.7 due to usage of async keyword
Build fails due to usage of async keyword as function name:
```
% python3.7 setup.py build
Traceback (most recent call last):
  File "setup.py", line 126, in <module>
    setup(**setup_kwargs)
  File "/usr/local/lib/python3.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/local/lib/python3.7/site-packages/setuptools/dist.py", line 370, in __init__
    k: v for k, v in attrs.items()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/dist.py", line 292, in __init__
    self.finalize_options()
  File "/usr/local/lib/python3.7/site-packages/setuptools/dist.py", line 529, in finalize_options
    ep.load()(self, ep.name, value)
  File "/Users/farrokhi/dev/psycopg2cffi/.eggs/cffi-1.11.5-py3.7-macosx-10.13-x86_64.egg/cffi/setuptools_ext.py", line 204, in cffi_modules
    add_cffi_module(dist, cffi_module)
  File "/Users/farrokhi/dev/psycopg2cffi/.eggs/cffi-1.11.5-py3.7-macosx-10.13-x86_64.egg/cffi/setuptools_ext.py", line 49, in add_cffi_module
    execfile(build_file_name, mod_vars)
  File "/Users/farrokhi/dev/psycopg2cffi/.eggs/cffi-1.11.5-py3.7-macosx-10.13-x86_64.egg/cffi/setuptools_ext.py", line 25, in execfile
    exec(code, glob, glob)
  File "psycopg2cffi/_impl/_build_libpq.py", line 171, in <module>
    _config = PostgresConfig()
  File "psycopg2cffi/_impl/_build_libpq.py", line 22, in __init__
    from psycopg2cffi import _config
  File "/Users/farrokhi/dev/psycopg2cffi/psycopg2cffi/__init__.py", line 4, in <module>
    from psycopg2cffi import extensions
  File "/Users/farrokhi/dev/psycopg2cffi/psycopg2cffi/extensions.py", line 39, in <module>
    from psycopg2cffi._impl import connection as _connection
  File "/Users/farrokhi/dev/psycopg2cffi/psycopg2cffi/_impl/connection.py", line 328
    def async(self):
            ^
SyntaxError: invalid syntax
```

### PR DESCRIPTION
**Pull request by farrokhi**
Rename `async` function and variable names to avoid conflict with reserved keyword (Fixes #99)

### CODE REVIEW

File: psycopg2cffi/_impl/connection.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def autocommit(self, value):
         self.set_session(autocommit=value)
 
     @property
-    def async(self):
+    def get_async(self):
```
[the remark this answers is not in the document]
Comment (farrokhi) [reply psycopg2cffi/_impl/connection.py]: Thanks for the hint. Totally makes sense. I will update the patch.

**Comment by farrokhi** [comment]
Python 3.7 support on Travis CI seems broken to me:
```
3.7 is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.7.tar.bz2
$ curl -sSf -o python-3.7.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
```

**Comment by farrokhi** [comment]
@lopuhin  Travis CI does not support Ubuntu Xenial worker yet. And python3.7 images are not available on Trusty. This causes the build failure message. Do you think we should disable 3.7 target for now?

**Comment by lopuhin** [comment]
> Do you think we should disable 3.7 target for now?

@farrokhi yes, let's disable 3.7 on travis for now and handle this separately - I forgot about this issues, sorry.

**Pull request merged**